site/docs/spark-structured-streaming.md

```diff
@@ -24,7 +24,21 @@ As of Spark 3.0, DataFrame reads and writes are supported.
 
 | Feature support                                  | Spark 3.0| Spark 2.4  | Notes                                          |
 |--------------------------------------------------|----------|------------|------------------------------------------------|
-| [DataFrame write](#writing-with-streaming-query) | ✔        | ✔          |                                                |
+| [DataFrame write](#writing-with-streaming-query) | ✔        | ✔         |                                                |
+
+## Streaming Reads
+
+Iceberg supports processing incremental data in spark structured streaming jobs which starts from a historical timestamp:
+
+```scala
+val df = spark.readStream
+    .format("iceberg")
+    .option(SparkReadOptions.STREAM_FROM_TIMESTAMP, Long.toString(streamStartTimestamp))
+    .load("database.table_name")
+```
+
+!!! Note
+    Iceberg only supports reading data from append snapshots. Overwrite snapshots cannot be processed and will cause an exception, similarly, delete snapshots will cause an exception by default, but deletes may be ignored by setting `streaming-skip-delete-snapshots=true`.
```

Comment:
The "overwrite" sentence should end after "cause an exception" and "similarly" should start a new sentence. There's a clean break there.

Reply:
done